website/templates/coach_dashboard.html

```diff
@@ -1,10 +1,25 @@
-{% extends "base2.html" %}
```

Comment:
I don't really like how this drop-down looks. Let's just discard it for now

Reply:
ok